cfgov/cfgov/settings/base.py

```diff
@@ -593,6 +594,15 @@
     # The template will render for the front-end, but the sortable code is missing
     # and the table will not be sortable until cf-tables from CF 4.x is implemented
     'SORTABLE_TABLES': {},
+
+    # Serve housing counselor JSON and PDFs from S3.
+    # Access e.g. /find-a-housing-counselor/?zipcode=20001&s3=True
+    'HOUSING_COUNSELOR_S3': {
+        'parameter': 's3',
+    },
```

Comment:
Possibly, this [was removed](https://github.com/cfpb/cfgov-refresh/pull/3186/files#diff-ac27c5919c46548eada7a7388e726c59L585) in #3186.